src/mimefactory.rs

```diff
@@ -1840,30 +1841,6 @@ mod tests {
         );
     }
 
-    #[test]
-    fn test_wrapped_base64_encode() {
```

Comment:
What's the reasoning for removing this test? To me it would seem nicer to bring back `wrapped_base64_encode()` by extracting it from `build_avatar_file()`, so that it can be tested again.